neurons/validators/src/core/validator.py

```diff
@@ -156,7 +157,7 @@ def fetch_miners(self, subtensor: bittensor.subtensor):
         bittensor.logging.info("Found %d miners", "fetch_miners", "fetch_miners", len(miners))
         return miners
 
-    async def set_weights(self, miners, subtensor: bittensor.subtensor):
+    async def set_weights(self, miners, subtensor: bittensor.subtensor) -> bool:
         bittensor.logging.info(f"[set_weights] scores: {self.miner_scores}")
 
         if not self.miner_scores:
```

Comment:
@LegendWorrier we need to log this as well.

Reply:
f"[set_weights] scores: {self.miner_scores}" will be added in log_text?